dashboards/php/php.json

```diff
@@ -3,7 +3,7 @@
 	"description": null,
 	"pages": [
 	  {
-		"name": "PHP",
+		"name": "Dashboard",
```

Comment:
Would we mind having a more descriptive name here? When a user installs this dashboard, the tab in nr1 will get set to "Dashboard" and not give too much detail on _what_ the dashboard does.

Examples may be `PHP Overview`, `Metrics Overview`, etc.

Reply:
Thank you @josephgregoryii for the suggestion. I have replaced the "Dashboard" with "PHP".